docs/resources/Channel.md

```diff
@@ -536,6 +541,33 @@ A thread member is used to indicate whether a user has joined a thread or not.
 
 ** \* These fields are omitted on the member sent within each thread in the [GUILD_CREATE](#DOCS_TOPICS_GATEWAY/guild-create) event **
 
+### Default Reaction Object
+
+An object that specifies the emoji to use as the default way to react to a forum post. Exactly one of `emoji_id` and `emoji_name` must be set.
+
+###### Default Reaction Structure
+
+| Field         | Type              | Description                                             |
+|---------------|-------------------|---------------------------------------------------------|
+| emoji_id      | snowflake         | the id of a guild's custom emoji, or 0 if unset         |
+| emoji_name    | ?string           | the unicode character of the emoji                      |
+
+### Forum Tag Object
+
+An object that represents a tag that is able to be applied to a thread in a `GUILD_FORUM` channel.
+
+###### Forum Tag Structure
+
+| Field       | Type      | Description                                                                                                    |
+|-------------|-----------|----------------------------------------------------------------------------------------------------------------|
+| id          | snowflake | the id of the tag                                                                                              |
+| name        | string    | the name of the tag (0-20 characters)                                                                          |
+| moderated   | boolean   | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
+| emoji_id    | snowflake | the id of a guild's custom emoji, or 0 if unset \*                                                             |
+| emoji_name  | ?string   | the unicode character of the emoji \*                                                                          |
+
+\* At most one of `emoji_id` and `emoji_name` may be set.
```

Comment:
Could this still be changed to be consistent with other emoji in the API?

```json
"emoji": {
  "id": 123,
  "name": "chillin",
  "animated": true
}
```

Reply:
Uh, not right now, I'm just documenting what's implemented, but feel free to file a feature request for it

Reply:
Done #5393